net.sourceforge.audmes.audmes.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id": "net.sourceforge.audmes.audmes",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "AudMeS",
+    "rename-icon": "audmes",
+    "rename-desktop-file": "audmes.desktop",
+    "finish-args": [
+        "--share=ipc",
+        "--filesystem=home",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio"
+    ],
+    "modules": [
+        {
+          "name": "wxWidgets",
+          "cleanup": [
+              "/bin",
+              "/include",
+              "/share/bakefile",
+              "/share/aclocal",
+              "/lib/wx/include"
+          ],
+          "config-opts": [
+              "--with-gtk=3",
+              "--disable-sdltest",
+              "--enable-unicode",
+              "--enable-display",
+              "--enable-propgrid",
+              "--disable-webkit",
+              "--disable-webview",
+              "--disable-webviewwebkit",
+              "--with-expat=builtin",
+              "--with-libiconv=/usr"
+          ],
+          "build-options" : {
+              "cxxflags":"-std=c++11"
+          },
+          "sources": [ 
+            {  
+                "type": "archive", 
+                "url": "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.5.1/wxWidgets-3.0.5.1.tar.bz2",
+                "sha1":"406ac736f61d88a3a866aa501e01e408a642c6e7"
```

Comment:
please don't use sha1. it is deprecated. we usually use sha256.

Reply:
checksum changed to sha256